ansible/roles/docker/templates/ubi81.Dockerfile.j2

```diff
@@ -25,7 +25,10 @@ RUN dnf install --disableplugin=subscription-manager -y \
       python3 \
       openssl-devel \
       procps-ng \
+      glibc-static \
     && dnf --disableplugin=subscription-manager clean all
+RUN git clone https://github.com/mhiramat/libcap && cd libcap && make \
+    make install
```

Comment:
```suggestion
RUN git clone https://github.com/mhiramat/libcap && cd libcap \
    && make install
```

Reply:
@danbev We've lost this fixup.

Reply:
Not sure how I did that but I've added it back now. Thanks